ansible_navigator/configuration/application_post_processor.py

```diff
@@ -0,0 +1,181 @@
+        return messages, errors
+
+    return wrapper
+
+
+class ApplicationPostProcessor:
+    """application post processor"""
+
+    @staticmethod
+    def _abs_user_path(fpath):
+        return os.path.abspath(os.path.expanduser(fpath))
+
+    @staticmethod
+    def _str2bool(value: Any) -> bool:
+        """convert some commonly used values
+        to a boolean
+        """
+        if isinstance(value, bool):
+            return value
+        if isinstance(value, str):
+            if value.lower() in ("yes", "true"):
+                return True
+            if value.lower() in ("no", "false"):
+                return False
+        raise ValueError
+
+    def _flatten_list(self, lyst: Any) -> List:
+        if isinstance(lyst, list):
+            return [a for i in lyst for a in self._flatten_list(i)]
+        return [lyst]
+
+    def _flatten_resolve_list_of_paths(self, value: List) -> List[str]:
+        value = self._flatten_list(value)
+        value = [self._abs_user_path(entry) for entry in value]
+        return value
+
+    def _true_or_false(self, entry: Entry, config: Config) -> Tuple[List[Message], List[str]]:
+        # pylint: disable=unused-argument
+        messages: List[Message] = []
+        errors: List[str] = []
+        try:
+            entry.value.current = self._str2bool(entry.value.current)
+        except ValueError:
+            errors.append(entry.invalid_choice)
+        return messages, errors
+
+    @staticmethod
+    @_post_processor
+    def cmdline(entry: Entry, config: Config) -> Tuple[List[Message], List[str]]:
+        # pylint: disable=unused-argument
+        """Post process cmdline"""
+        messages: List[Message] = []
+        errors: List[str] = []
+        if isinstance(entry.value.source, EntrySource):
+            if entry.value.source.name == "ENVIRONMENT_VARIABLE":
+                entry.value.current = entry.value.current.split()
+        return messages, errors
+
+    @_post_processor
+    def editor_console(self, entry, config) -> Tuple[List[Message], List[str]]:
+        """Post process editor_console"""
+        return self._true_or_false(entry, config)
+
+    @_post_processor
+    def execution_environment(self, entry, config) -> Tuple[List[Message], List[str]]:
+        """Post process execution_environment"""
+        return self._true_or_false(entry, config)
+
+    @_post_processor
+    def inventory(self, entry, config) -> Tuple[List[Message], List[str]]:
+        """Post process inventory"""
+        messages: List[Message] = []
+        errors: List[str] = []
+        if config.app == "inventory" and not entry.value.current:
+            msg = "An inventory is required when using the inventory subcommand"
+            errors.append(msg)
+            return messages, errors
+        if entry.value.current is Sentinel:
+            entry.value.current = []
+        else:
+            entry.value.current = self._flatten_resolve_list_of_paths(entry.value.current)
```

Comment:
This might will if hostname is given as input, since the path does not exist

Reply:
will look for a comma and not resolve as path